app/messages/messages/broker.rb

```diff
@@ -59,7 +62,10 @@ def bind_queue
     end
 
     def publish(message)
-      @exchange&.publish(message.payload, routing_key: bunny_config['routing_key'])
+      exchange.publish(message.payload, routing_key: bunny_config['routing_key'])
+    rescue StandardError => e
+      Rails.logger.error("Cannot publish to RabbitMQ.")
+      ExceptionNotifier.notify_exception(e)
     end
```

Comment:
I'm a bit concerned that this could get *very* chatty if we ever have an issue.

We could use error grouping to try and throttle the notifications a bit:
https://smartinez87.github.io/exception_notification/#error-grouping

Reply:
Also, the process is failing silently as far as the user is concerned. Do we want to show them a warning?

Reply:
And again, log the actual exception.

Reply:
> Also, the process is failing silently as far as the user is concerned. Do we want to show them a warning?

We decided not to show the users anything as it shouldnt stop their work as the messages can be resent later, so they can continue as normal as we get alerted and fix it

Reply:
> I'm a bit concerned that this could get _very_ chatty if we ever have an issue.
> 
> We could use error grouping to try and throttle the notifications a bit:
> https://smartinez87.github.io/exception_notification/#error-grouping

Added error grouping to deployment for LabWhere PR [here](https://github.com/sanger/deployment/pull/39)